packages/client/mutations/handlers/handleJiraCreateIssue.ts

```diff
@@ -13,19 +13,15 @@ const handleJiraCreateIssue = (payload: RecordProxy<any>, store: RecordSourceSel
   const teamMember = store.get(teamMemberId)
   const integrations = teamMember?.getLinkedRecord('integrations')
   const atlassian = integrations?.getLinkedRecord('atlassian')
-  const jiraSearchQueryId = toJiraSearchQueryId(meetingId)
-  const jiraSearchQuery = store.get(jiraSearchQueryId)
+  const meeting = store.get(meetingId)
+  const jiraSearchQuery = meeting?.getLinkedRecord('jiraSearchQuery')
```

Comment:
fixes a bug in which `const jiraSearchQuery = store.get(jiraSearchQueryId)` never returns anything

Reply:
_this_ is interesting! do you have a step to reproduce? I'd be curious to dig in & see why we can't access it directly